frontends/web/src/routes/exchange/exchange.module.css

```diff
@@ -94,4 +94,17 @@
 
 .updateButton {
     height: var(--item-height-small);
+}
+
+.infoContainer {
+    margin-top: var(--space-default);
+    .link {
+      color: var(--color-default);
+    }
```

Comment:
Vite uses PostCSS under the hood which supports nesting afaik, also CSS got native nesting in more modern engines (our webview uses currently smth like chrome 83, but this will prob change with Qt 6.2)
https://caniuse.com/css-nesting
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting

but we never used nesting in the codebase afaik and I don't think it should be necessary here so please try to make it not nested.